packages/components/src/color-palette/utils.ts

```diff
@@ -79,9 +92,9 @@ export const normalizeColorValue = (
 	value: string | undefined,
 	element: HTMLElement | null
 ) => {
-	const currentValueIsCssVariable = /^var\(/.test( value ?? '' );
+	const valueIsSimpleColor = value ? isSimpleCSSColor( value ) : false;
```

Comment:
I believe this change has created a regression.

It's treating `undefined` values as false positives = non-simple values, which means it then returns `defaultView?.getComputedStyle`

### With this changes

The default alpha value is `0`, which means, by default, any custom color cannot be seen in the canvas.

https://github.com/user-attachments/assets/3f4e9b8e-a65e-4536-a34d-664423d9abbd


### With these changes reverted

The alpha is `100` so, by default, my custom color changes are visible on the canvas.

https://github.com/user-attachments/assets/fb482ad7-f900-41f4-ac28-9b36915f81f7

Reply:
I have a fix here:

- https://github.com/WordPress/gutenberg/pull/64969

Would be good to confirm that it doesn't affect what this PR intends to do. Thanks!